teps/0122-complete-build-instructions-and-parameters.md

```diff
@@ -1,8 +1,8 @@
 ---
-status: implementable
+status: implemented
```

Comment:
please add the implementation prs to the references section of the TEP

Reply:
Thanks Jerop! I added the references section.